docs/how-to-guides/others/reducing-start-delays.md

```diff
@@ -0,0 +1,68 @@
+(a) when Autoware decides to make the ego vehicle start and
+(b) when the vehicle actually starts moving.
+More precisely:
+
+- (a) is the time when the speed or acceleration command output by Autoware switches to a non-zero value.
+- (b) is the time when the measured velocity of the ego vehicle switches to a positive value.
+
+## Start delay with manual driving
+
+First, let us look at the start delay when a human is driving.
+
+The following figure shows the start delay when a human driver switches the gear
+from _parked_ to _drive_ and instantly releases the brake to push the throttle pedal and make the velocity of the vehicle increase.
+
+![Manual driving delays](./images/start-delays/manual_driving.png)
+
+There are multiple things to note from this figure.
+
+- Brake (red): despite the driver instantly releasing the brake pedal,
+  we see that the measured brake takes around `150ms` to go from `100%` to `0%`.
+- Gear (orange): the driver switches gear _before_ releasing the brake pedal, but the gear is measured to switch _after_
+  the brake is released.
+- Throttle (green) and velocity (blue): the driver pushes the throttle pedal and the vehicle is measured to start moving around `500ms` later.
+
+## Gear delay
+
+In many vehicles, it is necessary to change gear before first starting to move the vehicle.
+When performed autonomously, this gear change can take some significant time.
+Moreoever, as seen from the data recorded with manual driving, the measured gear value may be delayed.
+
+In Autoware, the controller sends a stopping control command until the gear is changed to the _drive_ state.
+This means that delays in the gear change and its reported value can grealy impact the start delay.
+Note that this is only an issue when the vehicle is initially in the _parked_ gear.
+
+## Brake delay
+
+In vehicles with a brake pedal,
+the braking system will often be made of several moving parts which cannot move instantly.
+Thus, when Autoware sends brake commands to a vehicle,
+some delays should be expected in the actual brake applied to the wheels.
+
+This lingering brake may prevent or delay the initial motion of the ego vehicle.
+
+## Throttle response
+
+When pushing the throttle pedal, the wheels of the vehicle do not instantly start rotating.
+This is partly due to the inertia of the vehicle,
+but also to the motor which may take a significant time to start applying
+some torque to the wheels (see <https://en.wikipedia.org/wiki/Throttle_response>).
+
+Increasing the initial throttle may reduce the start delay but can cause uncomfortably high initial accelerations.
+
+## Conversion to actuation commands
+
+Depending on the vehicle interface, it may be necessary to convert the control command calculated by Autoware into
+an actuation message suitable for the low level vehicle controls.
+This conversion can be a source of delays if the initial acceleration of Autoware is not converted to a high enough actuation value.
```

Comment:
@maxime-clem 
I think I can understand what you mean thanks to your explanation🤔
But I couldn't understand this point may be because of lack of knowledge?

I imagined the description here is initial actuation value should be high to reduce start delay from the observation in the experiment. 
But are you saying that wrong conversion can be source of start delay in general?